packages/gatsby-telemetry/src/store.js

```diff
@@ -0,0 +1,76 @@
+const { homedir } = require(`os`)
+const path = require(`path`)
+const {
+  appendFileSync,
+  readFileSync,
+  renameSync,
+  existsSync,
+  unlinkSync,
+} = require(`fs`)
+const { ensureDirSync } = require(`fs-extra`)
+const Configstore = require(`configstore`)
+
+module.exports = class Store {
+  config = new Configstore(`gatsby`, {}, { globalConfigPath: true })
+
+  constructor() {}
+
+  getConfig(key) {
+    if (key) {
+      return this.config.get(key)
+    }
+    return this.config.all
+  }
+
+  updateConfig(...fields) {
+    this.config.set(...fields)
+  }
+
+  appendToBuffer(event) {
+    const bufferPath = this.getBufferFilePath()
+    appendFileSync(bufferPath, event, `utf8`)
+  }
+
+  getConfigPath() {
+    const configPath = path.join(homedir(), `.config/gatsby`)
```

Comment:
This isn't always guaranteed to be home dir since it falls back to `os.tmpdir()` (see comment above)

Could get path from config.path (https://github.com/yeoman/configstore#path)

Reply:
this should be ok for now